endpoints/authentication/src/main.cpp

```diff
@@ -396,6 +396,10 @@ namespace irods::http::handler
 					return _sess_ptr->send(std::move(res_rep));
 				});
 			}
+			else {
```

Comment:
indentation not ... quite right for the put above?   

consider putting this case at the top, since it's real short and easy to see scope?

Reply:
Hmm... could be...
```c++
if (!oidc_stanza_exists) {
    // this code
}
else if (did_except) {
    // the first block
}
else {
    // the middle block / existing else-if
}
```

I had a similar thought but wasn't sure if it would be worth doing.

Reply:
> indentation not ... quite right for the put above?

I'm not sure what you're referring to for "for the put above?". Also, clang-format hasn't reported an issue. Does the alignment look off to you?

> consider putting this case at the top, since it's real short and easy to see scope?

Will swizzle.

Reply:
this is for the 'get' case... so i was referring to the 'post' above, sorry, not 'put'.

Reply:
I thought about reordering things in the PR before this, but it was easy to do the bare minimum.

This whole source file is due for a refactor.

Reply:
Updated.